setup.py

```diff
@@ -156,30 +163,28 @@ def __init__(self, name):
 
 
 class OTIO_build_ext(setuptools.command.build_ext.build_ext):
-    # At the moment, we've only got code coverage set up for Linux, using gcov,
-    # which is specific to gcc.
-    if platform.system() == "Linux":
-        user_options = setuptools.command.build_ext.build_ext.user_options + [
-            (
-                'cxx-coverage',
-                None,
-                'Enable code coverage for C++ code.  NOTE: you will likely want to'
-                ' also set the build_tmp directory to something that does not get '
-                'cleaned up.',
-            )
-        ]
+    # At the moment, we've only got code coverage set up for Linux, using lcov,
+    # which is specific to llvm.
+    # if platform.system() in "Linux":
+    user_options = setuptools.command.build_ext.build_ext.user_options + [
+        (
+            'cxx_coverage',
+            None,
+            'Enable code coverage for C++ code.  NOTE: you will likely want to'
+            ' also set the build_tmp directory to something that does not get '
+            'cleaned up.',
+        )
+    ]
```

Comment:
@meshula whoops, I think we meant to delete this code, right?

Reply:
yes, our decision was to leave it to logic in cmake